### PR TITLE
Feature/theme 2

### PR DESCRIPTION
**Pull request by Tony---Zhang**
## Overview
1. Add dimensions
2. Add Icons
3. Refactor on Colors
4. Update samples
5. Fix dev menu issue

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## Out of scope
There is an issue on the Theme, we need to fix this later

## Checklist
1. [x] Makes your submission pass tests?
4. [x] Have you lint your code locally before submission?
6. [x] Have you updated the wiki of your change?

## Reference
- Card: You should add the card URL here.
- Related PR: The related pr of the current one.

| light | dark |
| -- | -- | 
| light | dark-theme |

### CODE REVIEW

File: ui-sample/src/main/java/com/thoughtworks/ark/uisample/SampleUIScreen.kt [hunk 1]
```diff
@@ -23,42 +35,144 @@ import com.thoughtworks.ark.uisample.state.NavigateActivityAction
 
 @Composable
 fun SampleUIScreen(
-    sendAction: (Action) -> Unit
+    sendAction: (Action) -> Unit,
```
Comment (yefeidd) [inline ui-sample/src/main/java/com/thoughtworks/ark/uisample/SampleUIScreen.kt:38]: Just a parameter, don't need the `,`

File: ui-sample/src/main/java/com/thoughtworks/ark/uisample/SampleUIScreen.kt [hunk 2]
```diff
@@ -23,42 +35,144 @@ import com.thoughtworks.ark.uisample.state.NavigateActivityAction
...
+private fun IconSamples() {
+    Row(modifier = Modifier.padding(top = Dimensions.standardSpacing)) {
+        IconSample(Color.Green, Color.Black)
+        Spacer(modifier = Modifier.width(Dimensions.standardSpacing))
+        IconSample(Color.Black, Color.White)
     }
 }
 
 @Composable
 private fun Item(
```
Comment (yefeidd) [inline ui-sample/src/main/java/com/thoughtworks/ark/uisample/SampleUIScreen.kt:91]: keep the `sendAction: (Action) -> Unit` as the latest parameter more make sense

File: ui/src/main/java/com/thoughtworks/ark/ui/themes/Theme.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package com.thoughtworks.ark.ui.themes
```
Comment (yefeidd) [inline ui/src/main/java/com/thoughtworks/ark/ui/themes/Theme.kt:1]: You also can add the Dimensions for here.

File: app/src/main/res/drawable-night/bottom_bar_color.xml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_checked="true" android:color="@color/purple_200" />
+    <item android:color="@color/grey700"  />
```
Comment (yefeidd) [inline app/src/main/res/drawable-night/bottom_bar_color.xml:4]: Whether we need to use the dynamic color here?

File: app/src/main/res/drawable/bottom_bar_color.xml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_checked="true" android:color="@color/purple_500" />
+    <item android:color="@color/grey700"  />
```
Comment (yefeidd) [inline app/src/main/res/drawable/bottom_bar_color.xml:4]: Whether we need to use the dynamic color?

**Review by yefeidd — APPROVED** [review]

**Review by jie-meng — APPROVED** [review]

**Pull request merged**